app/admin/RTMediaAdmin.php

```diff
@@ -195,34 +243,51 @@ function rtm_admin_templates() {
 				 * @param True to hide the option and false to show the option.
 				 */
 				$display = apply_filters( 'rtmedia_disable_media_in_commented_media', true );
-				if ( $display ) { ?>
+				if ( $display ) {
+					?>
 					<style type="text/css">
 						.rtm-option-wrapper .form-table[data-depends="buddypress_enableOnComment"] {
-						    display: none !important;
+							display: none !important;
 						}
 					</style>
 					<?php
 				}
 			}
 		}
 
-		function modify_medialibrary_permalink( $action, $post ) {
+		/**
+		 * Function to modify media library permalink.
+		 *
+		 * @param array  $action Action details array.
+		 * @param object $post Post Object.
+		 *
+		 * @return mixed
+		 */
+		public function modify_medialibrary_permalink( $action, $post ) {
 			$rtm_id = rtmedia_id( $post->ID );
 
 			if ( $rtm_id ) {
-				$link           = get_rtmedia_permalink( $rtm_id );
-				$title          = _draft_or_post_title( $post->post_parent );
-				$action['view'] = '<a href="' . esc_url( $link ) . '" title="' . esc_attr( sprintf( esc_html__( 'View &#8220;%s&#8221;', 'buddypress-media' ), $title ) ) . '" rel="permalink">' . esc_html__( 'View', 'buddypress-media' ) . '</a>';
+				$link  = get_rtmedia_permalink( $rtm_id );
+				$title = _draft_or_post_title( $post->post_parent );
+
+				// translators: 1. Title.
+				$action['view'] = '<a href="' . esc_url( $link ) . '" title="' . esc_attr( sprintf( 'View &#8220;%s&#8221;', $title ) ) . '" rel="permalink">' . esc_html__( 'View', 'buddypress-media' ) . '</a>';
```

Comment:
`esc_attr( sprintf( 'View "%s"', $title ) );` should work here, can we remove usage of html entity number in the current usage?

Reply:
Sure, will change and raise PR.